### PR TITLE
Add new payment method icon for Postfinance

### PR DESCRIPTION
**Pull request by larimaza**
## Purpose

Add new payment method icon for Postfinance.

## Definition of done

* [x] Development completed
* [x] Reviewers assigned

### CODE REVIEW

**Comment by larimaza** [comment]
@robinmetral Could you please check if the SVG file is ok? I don't know how to use Figma properly 😬 But I got this from the official Circuit UI Figma file.

Also, the Open Source check is failing but the link for me to accept it doesn't work, it keeps asking me to login with GitHub even though I already did!

File: packages/icons/manifest.json [hunk 1]
```diff
@@ -370,6 +370,11 @@
       "category": "Payment method",
       "size": "24"
     },
+    {
+      "name": "postfinance",
```
Comment (robinmetral) [inline packages/icons/manifest.json:374]: Let's make this `post_finance` so it ends up being `<PostFinance />` (just like `pay_pal`) 🙂 

```suggestion
      "name": "post_finance",
```

**Review by robinmetral — COMMENTED** [review]
Thanks! Just a small suggestion 👇

**Comment by robinmetral** [comment]
Ah and your questions:

> Could you please check if the SVG file is ok? I don't know how to use Figma properly 😬 But I got this from the official Circuit UI Figma file and ran it by the SVGO optimizer.

It looks good to me! If you follow the [Contributing Icons](https://circuit.sumup.com/?path=/docs/introduction-contributing-contributing-icons--page) guide it should always be fine. svgo is also ran on SVGs automatically in the pre-commit hook 🙂 

> Also, the Open Source check is failing but the link for me to accept it doesn't work, it keeps asking me to login with GitHub even though I already did!

Yup, this is a known issue with our CLA bot 😞 but no need to worry about this, you're employed by SumUp so don't need to sign it.

File: packages/icons/manifest.json (hunk 1 above)
Comment (larimaza) [reply packages/icons/manifest.json]: Good catch, I didn't realize the brand name was camelcase haha done!

Comment (robinmetral) [reply packages/icons/manifest.json]: I just know this because it's a payment method from 🇨🇭 

brings back memories 😛

**Review by robinmetral — APPROVED** [review]
Looks good to me! The icon gets cropped on the [Storybook icons page preview](https://oss-circuit-ui-git-feature-postfinance-logo.sumup-vercel.app/?path=/docs/features-icons--page) (because of layout. constraints) but it's fine—these brand icons "just" have to have a height of 24 but their width can vary (as it does here)

The PostFinance icon is cropped on the Storybook icons page

**Comment by robinmetral** [comment]
For some reason GH thinks this is still missing a successful deploy preview (there is one), so I'll go ahead and merge for you.  The release should be on npm in a few mins after Changesets does its magic ✨

**Pull request merged**